gapic-generator-cloud/templates/cloud/wrapper_gem/_main.erb

```diff
@@ -0,0 +1,75 @@
+<%- assert_locals gem -%>
+<% @requires = capture do %>
+<%- if gem.needs_dashed_ruby_file? -%>
+require "<%= gem.name %>"
+<%- end -%>
+<% end %>
+<%- gem.services.each do |service| -%>
+##
+# Create a new client object for <%= service.name %>.
+#
+# By default, this returns an instance of
+# <%= gem.docs_link version: gem.default_version, class_name: "#{service.name}::Client" %>
+# for version <%= gem.default_version.capitalize %> of the API.
```

Comment:
feels like should be a sub-template

Reply:
Generally we've been using partials for places where we think there's a reasonable chance that a particular section will want to be overridden by a subgenerator. Partials are the "unit" over overriding templates. In this case, I don't expect that we will have a subgenerator under the wrapper generator. If we ever do, we can easily refactor to break out a partial at that time.